easybuild/easyconfigs/s/SPOOLES/SPOOLES-2.2-goolf-1.5.14.eb

```diff
@@ -0,0 +1,32 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Bart Verleye
+
+easyblock='ConfigureMake'
+
+name = 'SPOOLES'
+version = '2.2'
+
+homepage = 'http://netlib.sandia.gov/linalg/spooles'
+description = """ A sparse linear equation solver """
+
+toolchain = {'name': 'goolf', 'version': '1.5.14'}
+toolchainopts = {'openmp': False}
+
+sources = ['%(namelower)s.%(version)s.tgz']
+source_urls = ['http://netlib.sandia.gov/linalg/spooles']
+
+buildopts = ['lib']
+
+patches = ['spooles-%(version)s-makeInc.patch']
+
+skipsteps = ['configure','install'] 
```

Comment:
include comment here explaining why you skip the `install` step and enable `buildininstalldir`, i.e. because the object files and what-not are required for `Calculix` which depends on `SPOOLES`